src/chrome/chromeDebugAdapter.ts

```diff
@@ -220,6 +221,10 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
             });
         }
 
+        if (args.supportsMappingURLsToFilePaths) {
+            this._pathTransformer = new FallbackToClientPathTransformer(this._session);
```

Comment:
Do this at the top of the method so we are less likely to add some code working on pathTransformer before this.